Documentation/ceph-client-crd.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Client CRD
+weight: 3500
+indent: true
+---
+
+# Ceph Client CRD
+
+Rook allows creation and updating clients through the custom resource definitions (CRDs).
+For more information about user management and capabilities see the [Ceph docs](https://docs.ceph.com/docs/master/rados/operations/user-management/).
+
+## Use Case
+
+Use Client CRD in case you want to integrate Rook with with applications that are using LibRBD directly.
+For example for OpenStack deployment with Ceph backend use Client CRD to create OpenStack services users.
+
+Don't use Client CRD for Flex or CSI driver users, they are created automatically.
+
+## Creating Ceph User
+
+To get you started, here is a simple example of a CRD to configure a Ceph client with capabilities.
+
+```yaml
+apiVersion: ceph.rook.io/v1
```

Comment:
One more thing... Could you add this example to the `cluster/examples/kubernetes/ceph` folder? Maybe call it `ceph-client.yaml`. This makes it simpler for users to try out the examples when they match in this folder.